pages/textbook-beta/course/quantum-machine-learning-course.vue

```diff
@@ -0,0 +1,183 @@
+  learnt.`]
+
+  headerImg = '/images/textbook-beta/course/quantum-machine-learning-course/header.png'
+
+  backToTextbookHomeLink: GeneralLink = {
+    url: '/textbook-beta',
+    label: 'Home',
+    segment: {
+      cta: 'back-to-textbook-home', location: 'header'
+    }
+  }
+
+  startLearningCTA: GeneralLink = {
+    url: textbookBetaQuantumMachineLearningCourseStartLearningUrl,
+    label: 'Start learning',
+    segment: {
+      cta: 'quantum-machine-learning', location: 'course'
+    }
+  }
+
+  courses: Course[] = [
+    {
+      image: 'introduction.png',
+      label: 'Introduction',
+      segment: { cta: 'qml-introduction', location: 'course' },
+      url: textbookBetaQuantumMachineLearningCourseStartLearningUrl
+    },
+    {
+      image: 'parameterized-quantum-circuits.png',
+      label: 'Parameterized quantum circuits',
+      segment: { cta: 'parameterized-quantum-circuits', location: 'course' },
+      url: 'https://learn.qiskit.org/course/machine-learning/parameterized-quantum-circuits'
+    },
+    {
+      image: 'data-encoding.png',
+      label: 'Data encoding',
+      segment: { cta: 'data-encdoing', location: 'course' },
+      url: 'https://learn.qiskit.org/course/machine-learning/data-encoding'
+    },
+    {
+      image: 'training-parameterized-quantum-circuits.png',
+      label: 'Training parameterized quantum circuits',
+      segment: { cta: 'Describing quantum computers', location: 'course' },
+      url: 'https://learn.qiskit.org/course/machine-learning/describing-quantum-computers'
+    },
+    {
+      image: 'supervised-learning.png',
+      label: 'Supervised learning',
+      segment: { cta: 'supervised-learning', location: 'course' },
+      url: 'https://learn.qiskit.org/course/machine-learning/supervised-learning'
+    },
+    {
+      image: 'variational-classification.png',
+      label: 'Variational classification',
+      segment: { cta: 'variational-classification', location: 'course' },
+      url: 'https://learn.qiskit.org/course/machine-learning/variational-classification'
+    },
+    {
+      image: 'quantum-feature-maps-kernels.png',
+      label: 'Quantum feature maps and kernels',
+      segment: { cta: 'quantum-feature-maps-kernels', location: 'course' },
+      url: 'https://learn.qiskit.org/course/machine-learning/quantum-feature-maps-kernels'
+    },
+    {
+      image: 'quantum-generative-adversarial-networks.png',
+      label: 'Quantum generative adversarial networks',
+      segment: { cta: 'quantum-generative-adversarial-networks', location: 'course' },
+      url: 'https://learn.qiskit.org/course/machine-learning/quantum-generative-adversarial-networks'
+    },
+    {
+      image: 'project.png',
+      label: 'Project',
+      segment: { cta: 'project', location: 'course' },
+      url: 'https://learn.qiskit.org/course/machine-learning/project'
+    }
+  ]
+
+  quantumMachineLearningCoursePrerequisites: Prerequisite[] = [
```

Comment:
Not a blocker for me, since the naming conventions were set before you began this work.. but I'd love to see us as a team, trying to shorten some of these variable names... 

@eddybrando - i'd love to hear your thoughts

Reply:
For now I've shortened it to `qmlCoursePrerequisites`, hopefully that's a bit better.